plugins/chat/assets/stylesheets/common/common.scss

```
@@ -987,3 +1037,22 @@ html.has-full-page-chat {
 [data-popper-reference-hidden] {
   visibility: hidden;
 }
+
+.channels-list {
+  .chat-user-avatar {
+    img {
+      width: calc(var(--channel-list-avatar-size) - 2px);
+      height: calc(var(--channel-list-avatar-size) - 2px);
+    }
+  }
+
+  .chat-channel-title {
+    &__users-count {
+      width: var(--channel-list-avatar-size);
+      height: var(--channel-list-avatar-size);
+      padding: 0;
+      font-size: var(--font-up-2);
```

feels a tad too bit to me
```suggestion
      font-size: var(--font-up-1);
```